db/testing.py

```diff
@@ -77,3 +78,22 @@ def load_low_level_data(self, mbid):
         """
         with open(self.data_filename(mbid)) as json_file:
             db.data.submit_low_level_data(mbid, json.loads(json_file.read()), gid_types.GID_TYPE_MBID)
+
+    def load_fake_low_level_data(self, mbid):
```

Comment:
Ah, I understand now. It would be more explicit to have a separate parameter to this function which you could manually change each time you want a different submission.

This might mean you need a bit more code when adding these items to the database (you won't be able to use a for loop unless you add some extra data), but I think it would make it clearer what the method does.